rest-api/unicode_csv.py

```diff
@@ -0,0 +1,67 @@
+import csv
+import codecs
+import cStringIO
+
+# Writes CSV files with Unicode encoding. (Standard CSV libraries don't handle Unicode.)
+# Adapted from https://docs.python.org/2/library/csv.html
```

Comment:
I've used https://pypi.python.org/pypi/unicodecsv/0.9.0 in the past, for this. The functionality below is simple enough, so I don't feel strongly... but see note below.

Reply:
OK. The license there is a little funny -- not a standard license of a type I'm familiar with. I removed UnicodeReader, since I'm not sure we're going to need it; I think UnicodeWriter will work ok.

Reply:
The [license](https://github.com/jdunck/python-unicodecsv/blob/master/LICENSE) is a standard [BSD 2-Clause](https://opensource.org/licenses/BSD-2-Clause) -- but I don't mind where we've ended up here.